server-spi/src/main/java/org/keycloak/models/ActionToken.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+* Entity for generate-action-token service endpoint.
+* @author VerHext
+*/
+
+package org.keycloak.models;
+
+public class ActionToken {
```

Comment:
I would just avoid having this class but building the JSON response with the "token" claim.

Reply:
You're right. I have now removed the class.